src/components/ebay-filter-menu-button/index.js

```diff
@@ -59,20 +49,16 @@ module.exports = require('marko-widgets').defineComponent({
             .map(item => item.value);
     },
     handleButtonKeydown(e) {
-        eventUtils.handleEscapeKeydown(e, () => {
+        eventUtils.handleEscapeKeydown(e.originalEvent, () => {
             this.buttonEl.setAttribute('aria-expanded', false);
         });
     },
-    handleItemClick(e, itemEl) {
-        this.setCheckedItem(this.getItemElementIndex(itemEl), itemEl);
+    handleMenuChange(e) {
+        this.setCheckedItem(this.getItemElementIndex(e.el), e.el);
     },
-    handleItemKeydown(e, itemEl) {
-        eventUtils.handleActionKeydown(e, () => {
-            this.setCheckedItem(this.getItemElementIndex(itemEl), itemEl);
-        });
-
-        eventUtils.handleEscapeKeydown(e, () => {
-            this.buttonEl.setAttribute('aria-expanded', false);
+    handleItemKeydown(e) {
```

Comment:
Minor, but can we use destructuring in the function definition to define `el` and `originalEvent`?

Reply:
Old habits ...

Reply:
Updating.